sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/SchemaUtilTest.java

```diff
@@ -105,7 +109,9 @@ public void testToBeamSchema() throws SQLException {
             .addField("boolean_col", Schema.FieldType.BOOLEAN)
             .addField("char_col", LogicalTypes.fixedLengthString(JDBCType.CHAR, 255))
             .addField("date_col", LogicalTypes.JDBC_DATE_TYPE)
-            .addField("decimal_col", Schema.FieldType.DECIMAL)
+            .addField("decimal_col", LogicalTypes.numeric(0, 0)) // doubtful
```

Comment:
even it's not degradation, it's really suspicious. Should we add `precision==0` as magic condition into `SchemaUtil. beamLogicalNumericField ()`?